src/server_manager/web_app/ui_components/outline-intro-step.js

```diff
@@ -348,6 +369,11 @@ Polymer({
     },
   },
 
+  _openLinkFreeTier: '<a href="https://cloud.google.com/free/docs/gcp-free-tier#compute">',
+  _openLinkIpPrice: '<a href="https://cloud.google.com/vpc/network-pricing#ipaddress">',
+  _openLinkFreeTrial: '<a href="https://cloud.google.com/free/docs/gcp-free-tier/#free-trial">',
+  _closeLink: '<iron-icon icon=open-in-new></iron-icon></a>',
```

Comment:
You can do the external anchor with CSS. For example: https://waaz.xyz/adding-external-link-indicator-with-css/

Reply:
That pattern won't work here, because "open-in-new" isn't a simple image URL.

This pattern is consistent with the approach used in outline-manual-server-entry.js, so if we can find a CSS-based replacement, we should probably use it in both files.